benchmarks/bench.sh

```diff
@@ -288,7 +288,7 @@ data_tpch() {
     else
         echo " Copying answers to ${TPCH_DIR}/answers"
         mkdir -p "${TPCH_DIR}/answers"
-        docker run -v "${TPCH_DIR}":/data -it --entrypoint /bin/bash --rm ghcr.io/databloom-ai/tpch-docker:main -c "cp -f /opt/tpch/2.18.0_rc2/dbgen/answers/* /data/answers/"
+        docker run -v "${TPCH_DIR}":/data -it --entrypoint /bin/bash --rm ghcr.io/scalytics/tpch-docker:main  -c "cp -f /opt/tpch/2.18.0_rc2/dbgen/answers/* /data/answers/"
```

Comment:
Seems that the `dbgen` binary in the container is being used to generate the tbl files.  `dbgen` looks like C code [built with gcc](https://github.com/scalytics/TPCH-Docker/blob/main/data/tpch/2.18.0_rc2/dbgen/makefile#L175) so I think it would be good to avoid introducing that toolchain into this repo

That container image looks pretty niche though, so there's a risk it might disappear or move again

Reply:
Amusingly duckdb actually builds this data generator into its source code: https://github.com/duckdb/duckdb/blob/4d24f5c660a205bf22a7fd99e36efece798452c4/extension/tpch/dbgen/dbgen.cpp#L29

Reply:
Maybe we need a rust crate for `dbgen` 🦀

Reply:
That would be amazing 🙏 

There appears to be one already actually, though it doesn't appear to be tpch: https://crates.io/crates/dbgen

Although it seems like @andygrove  actually made one already: https://crates.io/crates/tpctools

Maybe we can use that?